src/WorkflowsRules/phinx.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$dotEnv = Dotenv\Dotenv::createImmutable(__DIR__);
+$dotEnv->load();
+
+return [
+    'paths' => [
+        'migrations' => 'storage/db/migrations',
```

Comment:
WorkflowsRules have a phinx.example , and remember use a variable for path complete the path of migration

Reply:
@rwhite27 test break

Reply:
> WorkflowsRules have a phinx.example , and remember use a variable for path complete the path of migration

??? @FredPeal